setup/03-keywords_and_variables_rule.py

```diff
@@ -13,27 +13,43 @@ class InvalidKeywordName(KeywordRule):
     severity = ERROR
 
     def apply(self, keyword):
-        if not keyword.name in ALLOWED_KEYWORDS:
+        if not keyword.name.title() in ALLOWED_KEYWORDS:
```

Comment:
In the last comment, I talked about introducing method to normalizing keyword names. This seems it could be shared between exercises?

Reply:
Good idea, created `normalize()` in `static.py`